src/styles/admin-core/content-blocks/_image-text-background.scss

```diff
@@ -0,0 +1,121 @@
+@use "sass:math";
+
+@use "src/styles/abstracts" as *;
+
+$component: "c-block-image-text-background";
+
+.#{$component} {
+	width: 100%;
+	display: grid;
+	grid-template-columns: repeat(5, 1fr);
+	position: relative;
+
+	@include respond-to("md") {
+		grid-template-rows: repeat(2, auto);
+		grid-template-columns: repeat(1, 1fr);
+	}
+
+	&__content-wrapper {
+		width: 100%;
+		height: 100%;
+		display: flex;
+		align-items: flex-start;
+		justify-content: center;
+		flex-direction: column;
+		padding: $spacer-1xl;
+		z-index: get-z-layer("foreground");
+
+		p {
+			padding: unset !important;
+		}
+	}
+
+	&__content,
+	&__heading {
+		margin-bottom: $spacer-sm !important;
+	}
+
+	&__button {
+		margin-top: $spacer-xs;
+	}
+
+	&__image-wrapper {
+		border: 1px solid red;
```

Comment:
Was this border requested? Probably used for testing no?

Reply:
yes, i'll remove it